libhb/decavcodec.c

```diff
@@ -867,7 +867,8 @@ static hb_buffer_t *copy_frame( hb_work_private_t *pv )
                                                  context->height,
                                                  context->pix_fmt,
                                                  w, h, AV_PIX_FMT_YUV420P,
-                                                 SWS_LANCZOS|SWS_ACCURATE_RND);
+                                                 SWS_LANCZOS|SWS_ACCURATE_RND,
+                                                 SWS_CS_DEFAULT);
```

Comment:
To clarify a bit.  When the source is RGB, it is actually converted to YUV here.  In cropscale, we will probably always be converting YUV to YUV and only scaling.  But would it hurt to use hb_ff_get_colorspace in both places for consistency?